packages/react/src/autocomplete-input/AutocompleteInput.tsx

```diff
@@ -19,12 +19,12 @@ export const AutocompleteInput = forwardRef<
   return (
     <Input
       addonAfter={
-        downshift.inputValue &&
+        downshift.selectedItem &&
         !disabled && (
           <Button
             appearance="subtle"
             aria-label="clear"
-            icon={downshift.inputValue && <IconX />}
+            icon={downshift.selectedItem && <IconX />}
```

Comment:
we can drop this conditional - since we already evaluate this when adding the addon itself
```suggestion
            icon={<IconX />}
```

Reply:
Squashed the commit vai, it was committed before.

Reply:
hmm - in that case we might want to switch back to a regular input - cause we don't want the cross to show up unless a value has been selected - whereas the regular search input will show cross whenever it has value

Reply:
or we could use the `{...(selected && { addonAfter: null })}` trick i suppose